compiler/main/driver.cpp

```diff
@@ -2041,8 +2041,11 @@ static void checkLLVMCodeGen() {
       USR_FATAL("CHPL_TARGET_COMPILER=llvm not yet supported for this architecture");
     }
 
-    if (fIncrementalCompilation)
-      USR_FATAL("Incremental compilation is not yet supported with LLVM");
+    if (fIncrementalCompilation) {
+      const char* flag = (fParMake ? "-j'/'--parallel-make" : "--incremental");
```

Comment:
If I have `--parallel-make --incremental`, this logic will only complain about the first one. When user removes it, it will again complain about `--incremental`. That seems a bit unfortunate, but no deal-breaker for this PR. IOW, I am OK with merging, but if it is a relatively quick fix (tiny bit more logic?) it would be good to add it.

Reply:
Unless I've missed something, I don't think we can (easily) distinguish between whether you throw both flags or just `--parallel-make` since throwing `--parallel-make` causes `--incremental` to be auto-thrown as well.  So if we printed simply due to `fIncrementalCompilation` being set, we'd always print it even if only `--parallel-make` is thrown.

Of course, we could store more state when we're parsing those flags, or re-check for them, or …  But that feels like a bigger change than worthwhile, esp. just before the code freeze.  (I'm also betting heavily that almost nobody uses or knows about `--incremental`).

I could also go back to the error message that always mentions all three flags, but that felt potentially confusing since (again), my hypothesis is that nobody really knows about or uses `--incremental` (which is a pretty poorly named flag, IMO…  I can never remember what it does by name).

Reply:
(I think I've missed you for today, so I'm going to merge and we can discuss more tomorrow if you are seeing something I'm not).